### PR TITLE
Start item cleanup/start dungeon items

### PR DESCRIPTION
**Pull request by lepelog**
still needs ui

### CODE REVIEW

File: gamepatches.py [hunk 1]
```diff
@@ -1552,40 +1552,67 @@ def add_startitem_patches(self):
...
-                    startflags.append(flag)
-            else:
-                startflags.append(flags[item])
-        return startflags
+        start_item_counts = Counter(
+            map(strip_item_number, self.placement_file.starting_items)
+        )
+        # health is calculated in quarter hearts
+        starting_health = 6 * 4
+        starting_health += start_item_counts.pop(HEART_CONTAINER, 0)
```
Comment (yannl35133) [inline gamepatches.py:1560]: Shouldn't this be multiplied by 4 if the unit is quarters ?

Comment (lepelog) [reply gamepatches.py:1560]: good catch

**Comment by CovenEsme** [comment]
Still need to give this a test but would it make sense to have the maps as starting items by default? Would be good for newer players to have the dungeon maps already to help with finding checks.

**Comment by lepelog** [comment]
> Still need to give this a test but would it make sense to have the maps as starting items by default? Would be good for newer players to have the dungeon maps already to help with finding checks.

It might make sense to add them to the startitems for the beginner presets

**Pull request merged**